win32ss/user/winsrv/usersrv/lang/tr-TR.rc

```diff
@@ -22,22 +22,22 @@ BEGIN
     PUSHBUTTON "&Şimdi Sonlandır", IDC_END_NOW, 78, 98, 43, 17
     LTEXT "Bu program yanıt vermiyor.", IDC_STATIC, 7, 7, 178, 8
     LTEXT "ReactOS'a dönmek ve programın durumunu denetlemek için İptal'e tıklayınız.", IDC_STATIC, 7, 26, 178, 16
-    LTEXT "Programı hemen kapatmayı seçerseniz, tüm saklanmamış veriyi yitireceksiniz. Programdan şimdi çıkmak için Şimdi Sonlandır'a tıklayınız.", IDC_STATIC, 7, 53, 178, 26
+    LTEXT "Programı hemen kapatmayı seçerseniz, tüm saklanmamış veriyi yitireceksiniz. Programdan şimdi çıkmak için ""Şimdi Sonlandır""'a tıklayınız.", IDC_STATIC, 7, 53, 178, 26
 END
 
 
 STRINGTABLE
 BEGIN
-    IDS_SEVERITY_SUCCESS        "Success"
-    IDS_SEVERITY_INFORMATIONAL  "System Information"
-    IDS_SEVERITY_WARNING        "System Warning"
-    IDS_SEVERITY_ERROR          "System Error"
+    IDS_SEVERITY_SUCCESS        "Başarı"
+    IDS_SEVERITY_INFORMATIONAL  "Sistem Bilgisi"
+    IDS_SEVERITY_WARNING        "Sistem Uyarısı"
+    IDS_SEVERITY_ERROR          "Sistem Hatası"
 END
 
 STRINGTABLE
 BEGIN
-    IDS_SYSTEM_PROCESS          "System Process"
-    IDS_UNKNOWN_SOFTWARE_EXCEPT "unknown software exception"
-    IDS_OK_TERMINATE_PROGRAM    "Click on OK to terminate the program."
-    IDS_CANCEL_DEBUG_PROGRAM    "Click on CANCEL to debug the program."
+    IDS_SYSTEM_PROCESS          "Sistem İşlemi"
+    IDS_UNKNOWN_SOFTWARE_EXCEPT "bilinmeyen yazılım istisnası"
+    IDS_OK_TERMINATE_PROGRAM    "Programı sonlandırmak için Tamam'a basınız."
```

Comment:
Does this string (and the next one) mentions the names of the buttons to be clicked for performing the actions?